src/configuracion.ts

```diff
@@ -0,0 +1,21 @@
+import * as dotenv from 'dotenv';
+
+dotenv.config(); // Cargar variables de entorno desde el archivo .env
+
+export class Configuracion {
+    private logLevel: string;
+    private logFilePath: string;
+
+    constructor() {
+        this.logLevel = process.env.LOG_LEVEL || 'info';
+        this.logFilePath = process.env.LOG_FILE_PATH || './logs.log';
+    }
+
+    get LogLevel(): string {
+        return this.logLevel;
+    }
+
+    get LogFilePath(): string {
```

Comment:
Yo no sé lo que tenéis con los getters, de verdad...

Reply:
Solucionado en https://github.com/carlotiii30/organizacionSemanal/pull/69/commits/9c93c35010208b4eaca486dc136580c5610c2614